util/test/send_email.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
```

Comment:
I took a closer look at send_mail.py and email.mime.text (which happily was much simpler than I expected.) 

Looks good